guides/common/modules/con_package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,27 @@
+[id="package-mode-and-image-mode-hosts_"]
+= Package mode and image mode hosts
+
+{Project} identifies an image mode host as any system that reports a container image from its bootc status through Subscription Manager. 
+It classifies all other hosts as package mode hosts.
+
+Image mode simplifies enterprise operations by enabling development, operations, and solution providers to use the same container-native tools and techniques to manage both applications and the underlying operating system.
+
+In image mode, users can install and manage packages, but with limitations. 
+Any packages installed on image mode hosts are temporary and disappear after a reboot. 
+Because of this, installing packages is generally discouraged, except for debugging purposes or, in some cases, when applying critical security patches.
+
+You can display a column in the New UI of the *All Hosts* page that includes a mode indicator. 
```

Comment:
This is a late code change, but the column is going to be called "Type" instead of having the image mode icon. https://github.com/Katello/katello/pull/11319

Reply:
This is worded without specifically mentioning the name of the column so we could leave it as-is if necessary.